contrib/demo/runDemoScripts.sh

```diff
@@ -42,7 +46,7 @@ source ${DEMOS_DIR}/.startUtils
 setupTraps $0
 
 cleanupDemoOutput() {
-  sed -E --posix -e 's/\x1b\[(3J|H|2J)//g' -e '/^☸️ \$ /d' -e '/^ +certificate-authority-data: /d' -e '/^ +server: https:\/\/127\.0\.0\.1:[0-9]{5}$/d' -e 's/[0-9]+(s|m)/10s/g' -e 's/^(.*)( +[0-9]+s|[0-9]+m([0-9]+s)?)$/\1/' $1
+  sed -E -e 's/\x1b\[(3J|H|2J)//g' -e '/^☸️ \$ /d' -e '/^ +certificate-authority-data: /d' -e '/^ +server: https:\/\/127\.0\.0\.1:[0-9]{5}$/d' -e 's/[0-9]+(s|m)/10s/g' -e 's/^(.*)( +[0-9]+s|[0-9]+m([0-9]+s)?)$/\1/' $1
```

Comment:
Will need someone on Linux to verify the removal of `--posix` doesn't break things here.

Reply:
tested it and it doesn't break anything.